td-reproducible-tool/src/main.rs

```diff
@@ -140,6 +140,7 @@ fn main() -> std::io::Result<()> {
         println!(
             "INFO: -s or --strip_path is not specified, Skipping strip related rust file path."
         );
+        fs::write(binary_path, buf).expect("Failed to open the compiled binary!\n");
```

Comment:
This should be a bug fix, not related to rollback.

Please create another patch for this specific issue. Dont mix them.

Reply:
> This should be a bug fix, not related to rollback.
> 
> Please create another patch for this specific issue. Dont mix them.

Ok, Got it.